### PR TITLE
add stats from database, style changes

### PR DESCRIPTION
**Pull request by PTR-KLK**
- Code refactored
- Styling changes
- Made card container universal

TODO:
- Finish about card

### CODE REVIEW

File: src/pages/Home/Home.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { Header, Grid, Statistic } from "semantic-ui-react";
+import CustomBarChart from "../../Components/Home/CustomBarChart";
+import CustomPieChart from "../../Components/Home/CustomPieChart";
+// import CustomRadarChart from "../../Components/Home/CustomRadarChart";
```
Comment (HKrzyszczak) [inline src/pages/Home/Home.js:5]: If you don't need this please remove.

File: src/Components/home/ChartCard.component.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Card, CardContent } from "semantic-ui-react";
+
+function ChartCard(props) {
+  return (
+    <Card fluid>
+      <Card.Content>
+        {props.header ? (
+          <Card.Header textAlign="center">{props.header}</Card.Header>
+        ) : null}
```
Comment (cytrowski) [inline src/Components/home/ChartCard.component.js:10]: You can replace it with 
```
{
  props.header &&
  <Card.Header textAlign="center">{props.header}</Card.Header>
}
```

File: src/Components/home/ChartCard.component.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        <CardContent textAlign={props.textAlignContent}>
+          {props.children}
+        </CardContent>
+        {props.meta ? (
+          <Card.Meta textAlign="center">{props.meta}</Card.Meta>
+        ) : null}
```
Comment (cytrowski) [inline src/Components/home/ChartCard.component.js:16]: As above :)

File: src/Components/home/CustomBarChart.component.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  XAxis,
+  YAxis,
+  Tooltip,
+  CartesianGrid,
+  ResponsiveContainer,
+} from "recharts";
+
+function CustomBarChart({ barData }) {
+  return (
+    <div style={{ width: "100%", height: "40vh" }}>
```
Comment (cytrowski) [inline src/Components/home/CustomBarChart.component.js:14]: This should be put in CSS or in const above the component definition

File: src/Components/home/CustomPieChart.component.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  Cell,
+  Legend,
+  ResponsiveContainer,
+} from "recharts";
+
+const colors = ["#51d597", "#d5518f", "#5551d5"];
+
+function CustomPieChart({ pieData }) {
+  return (
+    <div style={{ width: "100%", height: "40vh" }}>
```
Comment (cytrowski) [inline src/Components/home/CustomPieChart.component.js:15]: Same story here - put it in CSS or in const before the function

File: src/Components/home/CustomPieChart.component.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      <ResponsiveContainer>
+        <PieChart>
+          <Pie dataKey="value" data={pieData} fill="#8884d8" label>
+            {pieData.map((entry, index) => (
+              <Cell
+                key={`cell-${index}`}
```
Comment (cytrowski) [inline src/Components/home/CustomPieChart.component.js:21]: No need to prefix it with `cell` - the `key` does not need to be unique per page - only per closest parent.

File: src/pages/home/Home.js [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+            mobile={14}
+            tablet={12}
+            computer={7}
+            largeScreen={6}
+            stretched
+          >
+            <ChartCard textAlignContent="center">
+              <Statistic>
+                <Statistic.Value className="home__statistic">
+                  {maxDistance + " km"}
```
Comment (cytrowski) [inline src/pages/home/Home.js:89]: Why not just `{maxDistance} km`?

**Pull request merged**